packages/multi-test/src/test_helpers/contracts/payout.rs

```diff
@@ -61,7 +61,10 @@ fn execute(
         .add_attribute("action", "payout"))
 }
 
-fn sudo(deps: DepsMut, _env: Env, msg: SudoMsg) -> Result<Response, StdError> {
+fn sudo<C>(deps: DepsMut, _env: Env, msg: SudoMsg) -> Result<Response<C>, StdError>
```

Comment:
This is a nice way to make it more generic.
For test contracts we can do it a lot.

(In real contracts we need to specify C in the function signature, so we can compile down to a concrete wasm `#[entry_point]`. Good to show we can handle ones that don't return that as well.